### PR TITLE
Finishing touches on dcrlnd cli pages.

### LINKED ISSUE #1060: Consider adding dcrlnd flags/args/commands
We do the same for dcrd and dcrwallet

https://docs.decred.org/wallets/cli/dcrd-and-dcrwallet-cli-arguments/
￼
its nice to make this stuff available on google search etc

### PR DESCRIPTION
**Pull request by jholdstock**
Includes the missing redirection from #1055 

Closes #1060

### CODE REVIEW

File: docs/lightning-network/dcrlncli/newaddress.md [hunk 1]
```diff
@@ -7,5 +7,4 @@
 
 ### Description
    
-Generate a wallet new address. Address-types has to be:
-- p2pkh: Pay to public key hash
+Generate a wallet new address. The only supported `address-type` is `p2pkh` (Pay to public key hash).
```
Comment (s-ben) [inline docs/lightning-network/dcrlncli/newaddress.md]: It seems the convention is to capitalize 'Pay To Public Key Hash'. In devdocs we use 'Pay-to-PubkeyHash'.

File: docs/wallets/cli/startup-basics.md [hunk 2]
```diff
@@ -49,6 +49,6 @@ The above example would first look to the `dcrd` configuration file for settings
 [Full List of Options for Each Application](dcrd-and-dcrwallet-cli-arguments.md)
 
 
-## Next:
+## Next
```
Comment (s-ben) [inline docs/wallets/cli/startup-basics.md]: Looks good, just wanted to make sure you intended to put this edit in this PR, as it's dcrd not dcrlnd.

Comment (jholdstock) [reply docs/wallets/cli/startup-basics.md]: Yep confirm this is deliberate. After [this review](https://github.com/decred/dcrdocs/pull/1055#discussion_r372317604), I ran a grep for headings which were ending with a colon and this one came up.

File: docs/lightning-network/dcrlncli/newaddress.md (hunk 1 above)
Comment (jholdstock) [reply docs/lightning-network/dcrlncli/newaddress.md]: Well spotted, will fix this

File: docs/lightning-network/dcrlncli/args-and-commands.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ requested RPC method, prints the response, and then terminates immediately.
 |`--rpcserver value`|host:port of Decred LN daemon (default: "localhost:10009")|
 |`--dcrlnddir value`|path to dcrlnd's base directory|
 |`--tlscertpath value`|path to TLS certificate|
-|`--chain value`, `-c value`|the chain lnd is running on e.g. decred (default: "decred")|
+|`--chain value`, `-c value`|the chain dcrlnd is running on e.g. decred (default: "decred")|
```
[the remark this answers is not in the document]
Comment (jholdstock) [reply docs/lightning-network/dcrlncli/args-and-commands.md]: I did actually notice this but I hoped nobody else would spot it. I couldn't be bothered to write the sed command to fix it.

**Comment by jholdstock** [comment]
I've implemented all your suggestions and rebased this on #1069 so we can get a successful build. Ready for another look

**Pull request merged**